src/sentry/migrations/0061_drop_eventattachment_index_without_file.py

```diff
@@ -0,0 +1,50 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.28 on 2020-04-06 16:46
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+    # This flag is used to mark that a migration shouldn't be automatically run in
+    # production. We set this to True for operations that we think are risky and want
+    # someone from ops to run manually and monitor.
+    # General advice is that if in doubt, mark your migration as `is_dangerous`.
+    # Some things you should always mark as dangerous:
+    # - Large data migrations. Typically we want these to be run manually by ops so that
+    #   they can be monitored. Since data migrations will now hold a transaction open
+    #   this is even more important.
+    # - Adding columns to highly active tables, even ones that are NULL.
+    is_dangerous = True
+
+    # This flag is used to decide whether to run this migration in a transaction or not.
+    # By default we prefer to run in a transaction, but for migrations where you want
+    # to `CREATE INDEX CONCURRENTLY` this needs to be set to False. Typically you'll
+    # want to create an index concurrently when adding one to an existing table.
+    atomic = False
+
+
+    dependencies = [
+        ('sentry', '0060_add_file_eventattachment_index'),
+    ]
+
+    operations = [
+        migrations.SeparateDatabaseAndState(
+            database_operations=[
+                migrations.RunSQL(
+                    """
+                    DROP INDEX CONCURRENTLY "sentry_eventattachment_project_id_date_added_750bbee8_idx";
```

Comment:
Just want to confirm where we got this index name from? When I try deleting this index the migration generates a statement with `DROP INDEX "sentry_eventattachment_project_id_25d761e1f446d2ff_idx";`.

Weirdly, in production I see `sentry_eventattachment_project_id_25d761e1f446d2ff`. Not sure what the inconsistency is about here...

Reply:
it's the name of the index that i get when running `sentry django sqlmigrate sentry 0001`

Reply:
```
--
-- Alter index_together for eventattachment (1 constraint(s))
--
CREATE INDEX "sentry_eventattachment_project_id_date_added_750bbee8_idx" ON "sentry_eventattachment" ("project_id", "date_added");
```